fs/io_zenfs.cc

```diff
@@ -389,25 +398,32 @@ void ZoneFile::PushExtent() {
   extent_filepos_ = fileSize;
 }
 
+IOStatus ZoneFile::AllocateNewZone() {
+  Zone* zone;
+  IOStatus s = zbd_->AllocateIOZone(lifetime_, io_type_, &zone);
+
+  if (!s.ok()) return s;
+  if (!zone) {
+    return IOStatus::NoSpace("Zone allocation failure\n");
+  }
+  SetActiveZone(zone);
+  extent_start_ = active_zone_->wp_;
+  extent_filepos_ = fileSize;
```

Comment:
Out of curiosity: Is there a reason why it is `fileSize` rather than `file_size_`?

Reply:
Just a typo I think. It should be file_size_. I bet there are tools that can lint these kinds of deviations from code style. You know, in Rust the compiler will issue a warning if you camelCase struct members :rofl:

Reply:
For historical/no good reason :) Lets fix this in a follow up-patch

Reply:
Created an issue for that https://github.com/westerndigitalcorporation/zenfs/issues/120